src/commands/cmd_server.cc

```diff
@@ -497,15 +508,26 @@ class CommandClient : public Commander {
           *output = redis::RESP_OK;
       }
       return Status::OK();
+    } else if (subcommand_ == "reply") {
+      if (reply_mode_arg_ == "on") {
+        conn->SetReplyMode(Connection::ReplyMode::ON);
+      } else if (reply_mode_arg_ == "off") {
+        conn->SetReplyMode(Connection::ReplyMode::OFF);
+      } else if (reply_mode_arg_ == "skip") {
+        conn->SetReplyMode(Connection::ReplyMode::SKIP_ONCE_PENDING);
+      }
+      *output = redis::RESP_OK;
+      return Status::OK();
     }
 
-    return {Status::RedisInvalidCmd, "Syntax error, try CLIENT LIST|INFO|KILL ip:port|GETNAME|SETNAME"};
+    return {Status::RedisInvalidCmd, "Syntax error, try CLIENT LIST|INFO|KILL ip:port|GETNAME|SETNAME|REPLY"};
   }
 
  private:
   std::string addr_;
   std::string conn_name_;
   std::string subcommand_;
+  std::string reply_mode_arg_;
```

Comment:
maybe we can parse it in `Parse()` and store a `ReplyMode reply_mode_` here?

Reply:
> maybe we can parse it in `Parse()` and store a `ReplyMode reply_mode_` here?

Changes has been made in [this commit](https://github.com/apache/kvrocks/pull/2943/commits/ca8f0afae0e6e365b06bc8ec30cda5dbb7e292ab)